pyproject.toml

```diff
@@ -1,5 +1,8 @@
+[project]
+name = "OTAnalytics"
+
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools", "hatchling"]
```

Comment:
Why do we need hatchling?

Reply:
Actually we don't need it. I just picked it up in some basic pyrpoject.toml example without questioning it. My bad 😅 I'll remove it